### PR TITLE
DTSPB-1670: ordinance survey new api

### PR DESCRIPTION
**Pull request by elvianihuseyin**
### JIRA link (if applicable) ###
PRO-1670: ordinance survey new api
### Change description ###
PRO-1670: ordinance survey new api

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,3 +23,4 @@ git.properties.json
 /pacts/
 /mochawesome-report
 /config/dev.yaml
+/functional-output
```
Comment (sanjayparekh-hmcts) [inline .gitignore:26]: Why did you need to add this?

Comment (elvianihuseyin) [reply .gitignore:26]: if you run the end to end test, it produced the functional-output, and so i put it inside gitignore so it didnt keep showing up when i tried to do commit. (it's not related to this ticket, but i think it;s good to put it inside gitignore)

**Review by sanjayparekh-hmcts — APPROVED** [review]
